.vscode/DodgeballCP/DCPscript.js

```diff
@@ -0,0 +1,175 @@
+'use strict'
+const arrOfPeople = [
+    {
+      id: 2,
+      name: "Charles Young",
+      age: 55,
+      skillSet: "welding",
+      placeBorn: "Omaha, Nebraska"
+    },
+    {
+      id: 3,
+      name: "Judy Twilight",
+      age: 35,
+      skillSet: "fishing",
+      placeBorn: "Louisville, Kentucky"
+    },
+    {
+      id: 4,
+      name: "Cynthia Doolittle",
+      age: 20,
+      skillSet: "tic tac toe",
+      placeBorn: "Pawnee, Texas"
+    },
+    {
+      id: 5,
+      name: "John Willouby",
+      age: 28,
+      skillSet: "pipe fitting",
+      placeBorn: "New York, New York"
+    },
+    {
+      id: 6,
+      name: "Stan Honest",
+      age: 20,
+      skillSet: "boom-a-rang throwing",
+      placeBorn: "Perth, Australia"
+    },
+    {
+      id: 7,
+      name: "Mia Watu",
+      age: 17,
+      skillSet: "acrobatics",
+      placeBorn: "Los Angeles, California"
+    },
+    {
+      id: 8,
+      name: "Walter Cole",
+      age: 32,
+      skillSet: "jump rope",
+      placeBorn: "New Orleans, Louisiana"
+    },
+  ]
+  
+  const listOfPlayers = []
+  const blueTeam = []
+  const redTeam = []
+  // creating player class
+  class player {
+    constructor(name, age, skillSet){
+      this.name = name
+      this.age = age
+      this.skillSet = skillSet
+      this.placeBorn = this.placeBorn
+    }
+    // adding the class to the people of the array
+    addToArrayOfPeople() {
+      arrOfPeople.push(this)
+    }
+  }
+  // creating blue team class
+  class blueTeammate extends player {
```

Comment:
I would have expected a single Teammate Class, with the team attribute or color being passed in to specify if it is a blue or red team.

Also, did not see you instantiate the class anywhere? was looking for `new Player(..)` or ` new Teammate(..)`